app/src/main/java/io/intrepid/contest/screens/entrysubmission/entryname/EntryNameContract.java

```diff
@@ -0,0 +1,23 @@
+package io.intrepid.contest.screens.entrysubmission.entryname;
+
+import io.intrepid.contest.base.BaseContract;
+
+public class EntryNameContract {
+    interface View extends BaseContract.View {
+        void onContestNameRetrieved(String contestName);
+
```

Comment:
I like to pull these spaces but I'll let you two make the call on what you want to do.